### ISSUE TITLE
New site template/theme available?

### ISSUE THREAD

**Issue by orchardc** [OrchardCMS/OrchardCore#5727]
Is there any third party site template/theme that will support OrchardCore?

**Comment by hishamco** [comment]
I don't think your question is directly to OC source, but hopefully if someone knows a site that supports OC themes?

Nothing fancy you could have a look to our docs and see how you can create a theme using OC

**Comment by orchardc** [comment]
I do have a question to the OC source once start doing my own Liquid landing page. :)

1. In Visual Studio, under the /Views/Test/ folder of my project, I created a new liquid file: Index.liquid; and then add it to the route.

2. In the Test Controller, just added the following code:

        public async Task<IActionResult> Index()
        {
            return await Task.Run(() => View());
        }

3. Edit the index.liquid, put the OC landing page code (agency):
{% if Model.ContentItem.Content.Services.ContentItems.size > 0 %}
//Some code here.............
{% endif %}

4. **My question is, where to assign value to Model???** I know how to do it in MVC, but not in Liquid.

**Comment by hishamco** [comment]
You are not limited to use Liquid only, you can use Razor if you want

**Comment by sebastienros** [comment]
You can't use a Liquid template "directly" as a View template. You need to create the "Index.cshtml" one, and from it, invoke a new Shape that would use the WhateverYouWant.liquid file.

```
@await DisplayAsync (await New.WhateverYouWant.(Foo: "Bar", Age: 12));
```

**Comment by ns8482e** [comment]
Technically you can use liquid, for view template, sample is available in PR #5099
